StreamChatClient_v3/APIClient/Endpoints/Payloads/ChannelCreatePayload_Tests.swift

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChatClient_v3
+import XCTest
+
+class ChannelCreatePayload_Tests: XCTestCase {
+    func test_channelCreatePayloadInitPropogates() {
```

Comment:
I'm not sure this test is needed for such a simple struct. I'd suggest merging this with the JSON test.